Assets/_BForBoss/Scripts/Editor/HierarchyWindowGroupHeader.cs

```diff
@@ -0,0 +1,35 @@
+//-----------------------------------------------------------------------
+// <copyright file="HierarchyWindowGroupHeader.cs" company="PERIGON GAMES">
+//     Copyright (c) Perigon Games. 2020 All Rights Reserved
+// </copyright>
+//-----------------------------------------------------------------------
+using UnityEditor;
```

Comment:
We don't need this, It's already part of the project under our PerigonTools section

Reply:
Should I Import PerigonTools into this project in this PR then? or just delete this file for now?

Reply:
that's strange, does it not already have perigon tools? it should be in ur manifest file. 

But try to delete this file, and see if it still works

Reply:
My bad, it was there